utils/wasm-builder/src/lib.rs

```diff
@@ -167,6 +177,25 @@ impl WasmBuilder {
         Ok(matched_features)
     }
 
+    fn need_remap_path(&self, enabled_features: &[String]) -> bool {
+        let crate_info = CrateInfo::from_manifest(&self.wasm_project.original_manifest_path())
+            .expect("Infallible");
+
+        let debug_in_features = enabled_features
+            .iter()
+            .filter_map(|feature| crate_info.features.get(feature))
+            .flatten()
+            .any(|feature| feature == "gstd/debug");
+
+        let debug_in_dependencies = crate_info.dependencies.iter().any(|dependency| {
+            dependency.kind == DependencyKind::Normal
+                && dependency.name == "gstd"
+                && dependency.features.contains(&String::from("debug"))
+        });
```

Comment:
Could it be the `debug` feature is enabled via a transitive dependency like
`dapp` crate depends on some `dapp-lib` crate which in turn depends on `gstd` with the `debug` feature on